suit_generator/cli.py

```diff
@@ -28,9 +33,36 @@
 }
 
 
+def configure_cli_logging(log_file_name: str = None):
+    """
+    Configure logging for CLI.
+
+    :param log_file_name: log file name to be used (ovveride default log file name from logging.yaml)
```

Comment:
typo in "ovveride", should be "override"

Reply:
fixed